markdown/org/grading.md

```diff
@@ -7,4 +7,233 @@ hidden: true
+
+Im zweiten Prüfungszeitraum besteht die theoretische Prüfung aus einer digitalen Klausur mit
+90 Minuten Dauer.
+
+Es wird jeweils rechtzeitig bekanntgegeben, ob die Prüfungen wie bisher im Home-Office durchgeführt
+werden oder ob dafür Rechner der FH genutzt werden, d.h. eine Anwesenheit in den Räumen der FH am
+Campus Minden notwendig ist. Die Termine in den Prüfungszeiträumen werden rechtzeitig vom Prüfungsamt
+bekannt gegeben, für den Termin in der Vorlesungszeit siehe `[Fahrplan]({{< ref "/org/schedule" >}})`{=markdown}.
+:::
+
+
+### 3. Gesamtnote: Gewichteter Mittelwert der beiden Teilleistungen
+
+**60% Praxis, 40% Theorie**
+
+[[Hinweis Abgabeslots, Präsenz/Online]{.bsp}]{.slides}
+
+
+## Hinweise zum Praktikum
+
+### Bearbeitung, Teams
+
+Sie bearbeiten alle Aufgaben in festen Teams zu je **zwei Personen**. Jedes Team erarbeitet seine
+**eigene** Lösung.
+
+Wer Lösungen ganz oder teilweise von anderen Teams/Studierenden oder anderen Quellen übernimmt
+und als eigene Lösung ab-/ausgibt, begeht einen Täuschungsversuch mit entsprechenden Konsequenzen
+im Prüfungsverfahren.
+
+### Abgabe per Git-Repo und Vorstellung im Praktikum
+
+Die Abgabe der Lösungen erfolgt über einen Pull- bzw. Merge-Request in einem nicht öffentlich
+sichtbaren Git-Repo, welches wir Ihnen zu Beginn des Semesters einrichten. Dazu benötigen Sie
+entweder einen kostenfreien und ggf. anonymen GitHub-Account oder Sie können alternativ den
+Gitlab-Server im Software-Labor nutzen, den Sie aus dem FH-Netz bzw. über VPN erreichen und wo
+Sie sich mit Ihren normalen FH-Zugangsdaten einloggen können ("LDAP").
+
+Die Lösungen müssen jeweils **vor** dem Praktikum hochgeladen werden (ggf. Deadlines beachten!)
+**und** **im** jeweiligen Praktikum von allen Teammitgliedern vorgestellt werden. Ohne diese
+Rücksprache gibt es keine Punkte.
+
+### Punkte und formale Abzüge
+
+Für die sinnvolle Umsetzung der Aufgaben erhalten Sie die jeweils angegebenen (Teil-) Punkte.
+
+Lösungen, die sich nicht kompilieren und/oder nicht ausführen lassen können höchstens wenige
+Teilpunkte bekommen. Dito Lösungen, bei denen sich die Vorgabe-Tests (so vorhanden) nicht
+kompilieren und/oder ausführen lassen. Stellen Sie vor Abgabe sicher, dass Ihr Code keine fest
+codierten Dateien/Verzeichnisse enthält. Denken Sie daran, dass bei UNIX ebenso wie bei Java
+zwischen Groß- und Kleinschreibung unterschieden wird, dass Dateinamen und Klassennamen korrekt
+sein müssen, dass die angegebenen Methodensignaturen korrekt sein müssen usw. ...
+
+Die Codequalität zählt. Selbst wenn Ihre Lösung eine perfekte Funktionalität aufweist, gibt es
+Abzug für schlechte oder schlampige Programmierung. Dies betrifft u.a. fehlende/mangelhafte
+Dokumentation, Verletzung von Namenskonventionen, uneinheitliche Formatierung,
+schlechte Namen für Klassen/Methoden/Variablen/..., duplizierter Code, ...
+
+Konkret können Sie für die Nichteinhaltung der formalen Kriterien folgende Abzüge pro Abgabe
```

Comment:
Hier würde ich noch hinzufügen, dass das ganze immer erst gilt, wenn das Thema in der VL bearbeitet wurde. Sonst kriegen die noch alle Panikattacken.

Reply:
danke. done.